deepspeed/ops/adam/fused_adam.py

```diff
@@ -131,6 +124,8 @@ def step(self,
                 state = self.state[p]
                 # State initialization
                 if len(state) == 0:
+                    # DeepSpeed processes each subgroup a time, so we need to keep tracking step for each tensor separately
+                    state['step'] = 0
```

Comment:
state['step'] = group.get('step', 0)

Reply:
This looks like a good idea. I will also put a more detailed comment to explain it otherwise other readers may get confused.